anise-cli/src/main.rs

```diff
@@ -50,7 +53,9 @@ pub enum CliErrors {
 
 fn main() -> Result<(), CliErrors> {
     if var(LOG_VAR).is_err() {
-        set_var(LOG_VAR, "INFO");
+        unsafe {
+            set_var(LOG_VAR, "INFO");
```

Comment:
Is `set_var` unsafe now?

Reply:
Yes it's been marked unsafeZ in the 2024 edition of rust